app/assets/stylesheets/elements/_buttons.css.scss

```diff
@@ -0,0 +1,40 @@
+// Dash Panel Buttons
+.nav-pills > li > a{
+  color: $text-white;
+  background-color: $dark-gray;
+  width: 150px;
+  height: 49px;
+  text-align: center;
+  border-radius:0;
+
+  p {
+    width: 108px;
+    height: 20px;
+    font-family: $default-font-bold;
+    font-size: 15px;
+  }
+}
+
+.nav-pills > li.active > a,
+.nav-pills > li.active > a:hover,
+.nav-pills > li.active > a:focus {
```

Comment:
If we could get by with just a single class declaration on the `a` tag that would be preferred. I don't know if you are fighting CSS specificity here or not but it's always better to have flatter CSS class declarations that way you don't always have to fight specificity.